doc/http-api/backend_swagger.yml

```diff
@@ -187,30 +175,35 @@ paths:
                 description: the recipient of the message
                 type: string
                 format: JID
-              msg:
+              body:
                 description: the message text
                 type: string
       description: Send a chat message from one user to another.
       responses:
         200:
           description: Message was sent
-  /messages/{caller}/{other}/{limit}:
+  /messages/{me}/{with}:
```

Comment:
Also from the implementation I read it is possible to get all messages with all users. I don't see relevant spec in the doc. Do I miss sth?